mappings/net/minecraft/client/gui/screen/resourcepack/ResourcePackListWidget.mapping

```diff
@@ -3,22 +3,23 @@ CLASS net/minecraft/class_521 net/minecraft/client/gui/screen/resourcepack/Resou
 		FIELD field_19128 client Lnet/minecraft/class_310;
 		FIELD field_19129 screen Lnet/minecraft/class_519;
 		FIELD field_19130 resourcePackList Lnet/minecraft/class_521;
-		FIELD field_19131 packContainer Lnet/minecraft/class_1075;
+		FIELD field_19131 pack Lnet/minecraft/class_1075;
```

Comment:
`pack` or `packProfile`?

Reply:
it's a profile, but since actual packs can't be used here so just call it a pack